boards/st/nucleo_wl55jc/doc/nucleo_wl55jc.rst

```diff
@@ -231,8 +231,8 @@ Default Zephyr Peripheral Mapping:
 .. rst-class:: rst-columns
 
 - LPUART_1 TX/RX : PA3/PA2 (ST-Link Virtual Port Com)
-- I2C_2_SCL : PA12 (Arduino I2C)
-- I2C_2_SDA : PA11 (Arduino I2C)
+- I2C_3_SCL : PA13 (Arduino I2C)
```

Comment:
```suggestion
- I2C_3_SCL : PB13 (Arduino I2C)
```

Reply:
Woops of course. Thanks for paying attention!